Rhino.Licensing/LicenseGenerator.cs

```diff
@@ -148,4 +201,4 @@ private static XmlDocument CreateDocument(Guid id, string name, DateTime expirat
             return doc;
         }
     }
-}
\ No newline at end of file
+}

```

Comment:
Can you extract this adding of a newline change to its own maint commit along with the other styling only changes?

Reply:
Done!